### DISCUSSION TITLE
Fix the spam console logs for /skin command

### DISCUSSION THREAD

**Discussion by sakurawald** [Zrips/CMI#7244] [Enhancement]
When you use /skin [an un-exist name], then the command feedback will become very very lag.
and then, you console will spam an exception like: java.net.ConnectException: Connection timed out
this is bad.

**Comment by mrfloris** [comment]
Can you please give us the full output of your `/ver` and `/cmi version`, and the full command you run, and the full error / log from the console?

**Reply by sakurawald** [reply]
I have just create an Bug Reposrt on Issues.
You can see that about /skin command problem.

**Comment by mrfloris** [comment]
Found the bug report https://github.com/Zrips/CMI/issues/7256 tnx
